README.md

```diff
@@ -56,6 +56,10 @@ keyboard (STM32F072 based).
 
 ![KeySeeBee](https://raw.githubusercontent.com/TeXitoi/keyseebee/master/images/keyseebee.jpg)
 
+[Arisu handwired](https://github.com/help-14/arisu-handwired) using STM32F401.
```

Comment:
Are you using a WeAct MiniF4 https://github.com/WeActTC/MiniSTM32F4x1

If yes, please

```suggestion
[Arisu handwired](https://github.com/help-14/arisu-handwired) using [WeAct
MiniF4](https://github.com/WeActTC/MiniF4-STM32F4x1).
```

Reply:
Similar but not from WeAct, I got it from taobao: https://item.taobao.com/item.htm?spm=a1z0k.7385961.1997985097.d4918993.4c5d65d7iflWTt&id=600716181302&_u=t2dmg8j26111